### ISSUE TITLE
Drill down / up hierarchies using the API when querying image ?

### ISSUE THREAD

**Issue by YovelD** [divinorum-webb/tableau-api-lib#111]
![Hierarchies in Tableau](https://help.tableau.com/current/pro/desktop/en-us/Img/hierarchy2.gif)

Hi ! 

Is there any way to "click" the + or - icon to drill down or drill up a hierarchy ? I'd then query a screenshot of the view.

**Comment by divinorum-webb** [comment]
@YovelD I don't believe Tableau's REST API supports interacting with the visual component of hierarchies when hitting their endpoints. A possible workaround would be to publish two have two versions of the worksheet published, one with the expanded hierarchy and one without. Then you could dynamically query `expanded_version` and `collapsed_version` as needed. If you don't want end users to see those versions, publish them to a project locked by permissions that no user has permissions to engage with but that your API user account does have permissions to interact with. That's how I'd go about working around the limitations, at least! Hope it helps.
